client/landing/stepper/declarative-flow/flows/ai-site-builder/ai-site-builder.ts

```diff
@@ -137,7 +137,7 @@ const aiSiteBuilder: Flow = {
 						}
 
 						window.location.replace(
-							`${ siteURL }/wp-admin/site-editor.php?canvas=edit&referrer=${ AI_SITE_BUILDER_FLOW }${ promptParam }`
+							`${ siteURL }/wp-admin/site-editor.php?p=%2Fpage%2F&canvas=edit&referrer=${ AI_SITE_BUILDER_FLOW }${ promptParam }`
```

Comment:
This would be an incorrect path I think, or it would just show the pages list instead of the editor.

#3041 should fix this issue by redirecting server side.

Reply:
Yeah 3041 fixes it